examples/seismic/tutorials/05_staggered_acoustic.ipynb

```diff
@@ -360,19 +360,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 16,
+   "execution_count": 50,
    "metadata": {},
    "outputs": [],
    "source": [
-    "norm_p = norm(p)\n",
-    "assert np.isclose(norm_p, .35098, atol=1e-4, rtol=0)"
+    "norm_p = norm(p4)\n",
```

Comment:
that's the only line that needs change, you should edit tyhr ipynb as a text file and only change these two lines